.circleci/config.yml

```
@@ -205,7 +205,7 @@ jobs:
           command: |
               source venv/Scripts/activate && pip install --no-cache-dir --upgrade -e . --progress-bar off && mkdir packages
               cd dash-renderer && renderer build && python setup.py sdist && mv dist/* ../packages/ && cd ..
-              git clone --depth 1 https://github.com/plotly/dash-core-components.git
+              git clone -b persistence-hg --depth 1 https://github.com/plotly/dash-core-components.git
```

Should be updated to point to the new branch?
```suggestion
              git clone -b hg-700-persistence --depth 1 https://github.com/plotly/dash-core-components.git
```